.github/instructions/Data.instructions.md

```diff
@@ -0,0 +1,27 @@
+---
+applyTo: "Source/Data/*"
+---
+
+These files define the mapping of phrases and words to their Bopomofo representations.
+
+In most cases, developers will add new Chinese characters or phrases rather than delete existing ones. When adding new characters or phrases, please perform the following checks:
+
+## Adding Phrases 
+
+When adding a new phrase containing multiple characters to `BPMFMappings.txt`, ensure that `phrase.occ` is also updated with the same phrase and its frequency. The frequency should be a positive integer.
+
+## Sorting
+
+Both `BPMFMappings.txt` and `phrase.occ` must stay sorted using the C locale. After making changes, run `LC_ALL=c sort -o BPMFMappings.txt BPMFMappings.txt` and `LC_ALL=c sort -o phrase.occ phrase.occ` before committing.
```

Comment:
issue: the lower case works, but it actually should be `LC_ALL=C` as defined in `/usr/share/i18n/locales/C`. We should also update the wiki.